src/app/all-tests/page.jsx

```diff
@@ -0,0 +1,808 @@
+"use client"
+//http://localhost:3000/all-tests
+
+let ids = [];
+for (var i = 0; i < 2; i++) {
+  ids.push(i + 3);
+}
+const titleStyle = "text-4xl font-bold"
+const linkStyle = "flex flex-col items-center justify-center p-6 mb-4 bg-[#201f65] rounded-lg shadow-md w-full h-full flex-grow flex-1 min-w-max text-white text-center text-lg font-bold"
+const redDivStyle = "text-white bg-red-500 p-[10px] mt-[20px] rounded font-bold min-h-[48px]"
+const blueButton = "bg-blue-500 text-white border border-black p-2 rounded max-w-xs w-full min-h-[48px]"
+const redButton = "bg-red-500 text-white  p-2 rounded max-w-xs w-full min-h-[48px]"
+
+const linkList = [
+  "aria-command-name",
+  "aria-deprecated-role",
+  "aria-meter-name",
+  "aria-required-attr",
+  "aria-required-parent",
+  "aria-toogle",
+  "aria-valid-attr-value",
+  "blink-tag",
+  "button-name",
+  "color-contrast",
+  "input-button-name",
+  "link-name",
+  "meta-refresh",
+  "meta-viewport",
+  "neurodiversity ruleset",
+  "scrollable-region-focusable",
+  "select-name",
+  "svg-img-alt",
+  "image-alt",
+  "aria-roles",
+  "aria-tooltip-name"
+];
+
+
+export default function AllTests() {
+
+  return (
+    <div className="flex flex-col items-center justify-start p-8 space-y-4 !bg-blue-500 children-test-container">
+      <h1 className={titleStyle}>Not Compliant tests</h1>
+      <div className="!grid grid-cols-3 gap-4 w-full p-4">
+        {linkList.map(item => {
+          return (
+            <a
+              key={item}
+              href={`#${item}`}
+              className={linkStyle}
+            >
+              <p>{item}</p>
+            </a>
+          )
+        })}
```

Comment:
The grid can be updated to something like this so that it's more responsive: 


```suggestion
        <div className="!grid grid-cols-2 md:grid-cols-3 gap-4">
          {linkList.map(item => {
            return (
              <a
                key={item}
                href={`#${item}`}
                className={linkStyle}
              >
                <p className="w-20 sm:w-36 lg:w-56">{item}</p>
              </a>
            )
          })}

        </div>
```